src/components/SearchBar/SearchBar.jsx

```diff
@@ -1,36 +1,74 @@
 import React from 'react';
 import styles from './SearchBar.module.css';
+import { loadData } from './connect-store';
+import { connect } from 'react-redux';
 
-export default function SearchBar() {
-    return (
-        <div className={styles.searchBarContainer}>
-            <h4 className={styles.title}>Find your movies</h4>
-            <input
-                className={styles.input}
-                placeholder="Enter movie title or genre"
-            />
-            <div className={styles.filterBarContainer}>
-                <div className={styles.leftSide}>
-                    <h5
-                        className={`${styles.leftSideItem} ${styles.searchTitle}`}
-                    >
-                        Search by
-                    </h5>
-                    <button
-                        className={`${styles.leftSideItem} btn btn-primary`}
-                    >
-                        Title
-                    </button>
-                    <button
-                        className={`${styles.leftSideItem} btn btn-primary`}
-                    >
-                        Genre
-                    </button>
-                </div>
-                <div>
-                    <button className="btn btn-primary">SEARCH</button>
+class SearchBar extends React.Component {
+    switchByTitle() {
+        localStorage.setItem('filter', 'query');
```

Comment:
What the reason for using localStorage?

Reply:
We need to switch search based on filter. In future we will sort data based on filter. If we create variable inside this component, we couldn't use it with sorting.

Reply:
I think, that using of localstorage ismuch easier then using redux store, is not it?

Reply:
For using redux we need to create two more actions, two more action-creators, add additional cases in reducer and other... For using localstorage we need just one code-string. And with redux we should connect store to loadData function. Are we sure, that we want it?